modulefiles/GDAS/wcoss2.intel.lua

```diff
@@ -33,13 +33,17 @@ load("nco/5.0.6")
 load("gsl/2.7")
 load("prod_util/2.0.14")
 load("bufr/12.0.1")
+load("fms-C/2023.04")
 
 -- hack for pybind11
 setenv("pybind11_ROOT", "/apps/spack/python/3.8.6/intel/19.1.3.304/pjn2nzkjvqgmjw4hmyz43v5x4jbxjzpk/lib/python3.8/site-packages/pybind11/share/cmake/pybind11")
 
 -- hack for git-lfs
 prepend_path("PATH", "/apps/spack/git-lfs/2.11.0/gcc/11.2.0/m6b6nl5kfqngfteqbggydc7kflxere3s/bin")
 
+-- hack for FMS
+setenv('fms_DIR', '/apps/prod/hpc-stack/i-19.1.3.304__m-8.1.12__h-1.14.0__n-4.9.2__p-2.5.10__e-8.6.0pnetcdf/intel-19.1.3.304/cray-mpich-8.1.12/fms/2023.04/lib/cmake/fms/')
```

Comment:
```suggestion
setenv('fms_ROOT', '/apps/prod/hpc-stack/i-19.1.3.304__m-8.1.12__h-1.14.0__n-4.9.2__p-2.5.10__e-8.6.0pnetcdf/intel-19.1.3.304/cray-mpich-8.1.12/fms/2023.04')
```
`cmake` recommends using `_ROOT`

Reply:
@Hang-Lei-NOAA This is the FMS modulefile with the missing `fms_ROOT` variable.  Please let the WCOSS2 admins know to include this variable in this and all future FMS installations.